packages/graphql/src/nexus.ts

```diff
@@ -49,6 +60,12 @@ export const QuestionData = interfaceType({
     if (item.type === DB.QuestionType.SC || item.type === DB.QuestionType.MC) {
```

Comment:
this logic basically defines, how we choose which type of question options will be returned. we use the type of the question to decide.